### PR TITLE
Detail view controller scroll view

### PR DESCRIPTION
**Pull request by Slavitor**
Adding UIViewController with scrollView. ScrollView has three placeholder UIViews that will host subviews.
Adding UIViewController extension that will handle viewController presentation. All custom presentation code for UIViewControllers will be in this extension.

### CODE REVIEW

File: DVTShowcase/AppDetailViewController.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.title = selectedApp["name"] as? String
+
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
```
Comment (pjwelcome) [inline DVTShowcase/AppDetailViewController.swift]: remove this if you are not using it.

File: DVTShowcase/AppDetailViewController.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+    
+    init(with dvtApp:[String: AnyObject?]) {
```
Comment (pjwelcome) [inline DVTShowcase/AppDetailViewController.swift]: Why is this a dictionary? can you not just pass the fail why do you need to pass the whole dictionary to the next viewcontroller

**Review by pjwelcome — CHANGES_REQUESTED** [review]

**Review by AntagonistBob — APPROVED** [review]

File: DVTShowcase/AppDetailViewController.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    //MARK: Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.title = selectedApp.client
+    }
+
+    init(with dvtApp:DVTApp) {
+        selectedApp = dvtApp
+        print(dvtApp)
```
Comment (pjwelcome) [inline DVTShowcase/AppDetailViewController.swift:26]: removed print. Dont leave logging on anything in the code.

**Review by pjwelcome — APPROVED** [review]

**Comment by pjwelcome** [comment]
One last thing is that you didnt need to make 11 commits for this PR. Please try and keep it less commit as possible, just way to many for this little amount of code.

**Pull request merged**